src/gtk-3.20/scss/apps/_nemo.scss

```diff
@@ -18,7 +18,8 @@
 
     .nemo-window {
         .nemo-inactive-pane .view { 
-            background-color: mix($base_color, $text_color, .12);
+            background-color: darken($base_color, 5%); 
```

Comment:
darken is not working fine with both light and dark colorscheme, use mix instead